src/containers/AssetList.js

```diff
@@ -111,4 +112,8 @@ const AssetList = ({ match }) => (
   </div>
 );
 
-export default AssetList;
+const RoutedAssetList = () => (
+  <Route path="/assets/:filter" component={AssetList}/>
+);
+
+export default RoutedAssetList;
```

Comment:
Again, I would prefer having the Route be separate from the component. Maybe AssetList.js and RoutedAssetList.js? Or just leaving the Route in App.js?